querybook/webapp/components/DataDocCell/DataDocCell.tsx

```diff
@@ -80,6 +92,60 @@ export const DataDocCell: React.FunctionComponent<IDataDocCellProps> =
                 fullScreenCellIndex,
             } = useContext(DataDocContext);
 
+            let queryCells = useQueryCells(docId);
+            const queryTitle = queryCells[index].meta.title;
+            const title =
+                queryTitle == null || queryTitle === ''
+                    ? 'Query #' + (index + 1).toString()
+                    : queryTitle;
+            queryCells = queryCells.slice(index);
+
+            const latestQueryExecutions = useMakeSelector(
+                makeLatestQueryExecutionsSelector,
+                queryCells.map((c) => c.id) ?? []
+            );
+            const hasQueryRunning = useMemo(
+                () => latestQueryExecutions.some((q) => q.status < 3),
```

Comment:
use QueryExecutionStatus.DONE

btw i noticed this code is copy pasted from somewhere else, can you also fix that? 🙏